contrib/example.tf

```diff
@@ -0,0 +1,17 @@
+module "event-gateway" {
+  source = "terraform"
```

Comment:
Could you change it to full github URL so people can just copy/paste?

Reply:
sure, I'll do a little refactor and add a readme file when I'm done with the etcd module

Reply:
awesome thanks!